client/app/components/button/RequestAccessButton.tsx

```diff
@@ -55,14 +60,14 @@ export default function RequestAccessButton({
           </Alert>
         ))}
       <Button
-        style={{ margin: "0 auto", display: "flex" }}
-        sx={{ ...commonButtonConfig }}
-        aria-label="Request Access"
+        className="my-10"
+        sx={{ textTransform: "none" }} //to remove uppercase text
```

Comment:
wow, we didn't need any of that styling that was previously in here?

Reply:
We could keep them, I just tried to make it look identical to other buttons in our app.